ibm_mq/assets/configuration/spec.yaml

```diff
@@ -52,7 +52,9 @@ files:
           type: string
         description: <PASSWORD> of the IBMMQ Channel to connect to.
       - name: queues
-        description: The IBMMQ queues to get data from (no queue metrics will be sent, but the check will run without it).
+        description: |
+          The IBMMQ queues to get data from (no queue metrics will be sent, but the check will run without it).
+          This will be in addition to any queues discovered using autodiscovery, regex or patterns.
```

Comment:
```suggestion
          This will be in addition to any queues discovered using Autodiscovery, regex, or patterns.
```

Reply:
why caps? I'm regerring to the `auto_discover_queues ` config option.